todo-app/public/index.html

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" href="index.css">
```

Comment:
This is not the right way to include styles in your `create-react-app` project!

You can read [the docs here](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-a-stylesheet) about how styles should work within `create-react-app` projects.

We can also discuss this in person, but basically there is a relatively complicated build toolchain set up to allow you to make your styles modular, and to be able to import them into your js files in a logical way. The days of having a single massive `index.css` are over, we use modular styles now.

A great example is if you build a component, and then suddenly need to use this component in another project. We can simply copy/paste the entire component directory:

    |- MyComponent
      ---- MyComponent.js
      ---- MyComponent.styles.js

And not have to worry about messy global styles.